playbooks/adhoc/noc/get_zabbix_problems.yml

```diff
@@ -2,6 +2,8 @@
 - name: 'Get current hosts who have triggers that are alerting by trigger description'
   hosts: localhost
   gather_facts: no
+  roles:
+    - os_zabbix
   tasks:
```

Comment:
If you're going to rely on the role to pull in the module, then you should change "tasks" to "post_tasks" to make it clear that it'll run after roles (more clarity than anything IMO).